src/Entity/City.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Entity;
+
+use ApiPlatform\Core\Annotation\ApiResource;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\SoftDeleteable\Traits\SoftDeleteableEntity;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
+use Symfony\Component\Serializer\Annotation\Groups;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Table(name="cities")
+ * @ORM\Entity(repositoryClass="App\Repository\CityRepository")
+ *
+ * @ApiResource(
+ *     attributes={
+ *         "normalization_context"={"groups"={"read"}},
+ *         "denormalization_context"={"groups"={"write"}}
+ *     },
+ *     collectionOperations={"get", "post"},
+ *     itemOperations={"get"}
+ * )
+ */
+class City
+{
+    use TimestampableEntity,
+        SoftDeleteableEntity;
+
+    /**
+     * @var int
+     *
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     *
+     * @Groups("read")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255, unique=true)
+     *
+     * @Assert\Length(min=3, max=100)
+     * @Groups({"read", "write"})
+     */
+    private $name;
+
+    /**
+     * @return mixed
```

Comment:
This method return only integer value, why in phpdoc wrote mixed value will return in there?